pkg/asset/manifests/network-operator.go

```diff
@@ -15,29 +15,29 @@ const (
 	defaultMTU = "1450"
 )
 
-// networkOperator generates the network-operator-*.yml files
-type networkOperator struct {
+// NetworkOperator generates the network-operator-*.yml files
```

Comment:
Why make it public?

Reply:
Just so that we can unmarshal it from outside the package. Its type is 'manifests.networkOperator' and to be able to call its constructor for unmarshal we need it to be exported.

Reply:
@rajatchopra any thoughts on keeping the struct private?

Reply:
Ack. Rolling the struct itself to be private.